src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts

```diff
@@ -262,6 +262,30 @@ export class SolutionExplorerList {
     }, 0);
   }
 
+  public getPartToDisplayOfSolutionUri(solutionUri: string): string {
```

Comment:
```suggestion
  public getSolutionName(solutionUri: string): string {
```

Please always consider, what a user of this public method wants to do.

Why should a user care about how the solution name is represented?
This is far more explicit and tells me, what I can do with this.